src/resources/postcss/views/_list.pcss

```diff
@@ -93,9 +93,12 @@
 	}
 
 	.tribe-events-calendar-list__event-title {
+		@mixin heading;
+		@mixin heading-6;
```

Comment:
I'm seeing only `h5` in the designs, we should add the `h5` class to the markup.

Reply:
If you inspect the designs, you will see that for mobile is 6 (16px) and for desktop, 5.

Reply:
Ah I assumed we had different mobile and desktop heading sizes for heading 5, by looking at the class names mobile says `mobile-heading-5` and desktop says `desktop-heading-5`. This is something we should check with matt/rachel on, if they named these classes this way by accident or it's intended. The typography sheet hasn't been updated either, so we should reconcile this.

Reply:
I think the problem is that, this time we didn't have a description of the changes, so I did a pass on the updated designs and updated accordingly.

Reply:
from the call we'll add `.tribe-common-h5--min-medium` and use that for the title.